packages/dashboard/src/components/MultiCountChart/MultiCountStyles.ts

```diff
@@ -0,0 +1,40 @@
+import { IMultiCountChartStyles, IMultiCountStyleProps } from './MutliCountChart.types';
+
+export const getStyles = (props: IMultiCountStyleProps): IMultiCountChartStyles => {
+  return {
+    root: {
+      display: 'flex',
+      lineHeight: '33px',
```

Comment:
Where is this `lineHeight: 33px` value coming from? It will likely truncate the descenders of the text.